### ISSUE TITLE
Bundler could not find compatible versions for gem "websocket-driver"

### ISSUE THREAD

**Issue by smrezvani** [NexusSW/lxd-common#29]
Hi,
I 'm new in rails and I want's to install lxd-common to my project. After add gem 'lxd-common' to my Gemfile and run bundle, I get this error:

Bundler could not find compatible versions for gem "websocket-driver":
  In snapshot (Gemfile.lock):
    websocket-driver (= 0.6.5)

  In Gemfile:
    rails (~> 5.1.5) was resolved to 5.1.5, which depends on
      actioncable (= 5.1.5) was resolved to 5.1.5, which depends on
        websocket-driver (~> 0.6.1)

    lxd-common was resolved to 0.9.8, which depends on
      nio4r-websocket (~> 0.6) was resolved to 0.6.2, which depends on
        websocket-driver (~> 0.7)

I installed all dependencies with "gem install ...." but nothing changed.
Ruby version: 2.5.0p0
Rails version: 5.1.5

**Comment by thewyzard44** [comment]
Good timing!!  Documenting this gem has recently bubbled up very high on my todo list...  I'll try to get some starter documentation up this evening to make your life a bit easier.

In regards to your issue, it appears that either rails, or one of your other dependencies has locked the websocket-driver gem to version 0.6.5 whereas my nio4r-websocket gem specifies ~> 0.7.  I did that for no good reason other than it was the current version at the time that I wrote the gem.

And so I'll attempt to relax that restriction to allow 0.6.5, run it through some tests, and let you know 😃
